### PR TITLE
ngc_rdma_wrapper changes

### PR DESCRIPTION
**Pull request by RavidRothfeld**
- Align ngc_rdma_wrapper to the internal_lb wrapper
- Some cosmetic changes to internal_lb wrapper

### CODE REVIEW

File: ngc_rdma_wrapper.sh [hunk 1]
```diff
@@ -178,14 +351,38 @@ if [[ $# == 1 ]]; then
...
     check_ssh "${SERVER_IP}" "${CLIENT_IP}"
-    # Get MLNX devices (PCIe & RDMA):
-    ssh "${SERVER_IP}" sudo dmidecode -t 1 | grep -i serial | awk '{$1=$1};1' | grep -iv '^$' &>> "${LOGFILE}"
-    ssh "${SERVER_IP}" sudo dmidecode -t 0 | grep -i version | awk '{$1=$1};1' | sed 's/^/BIOS /' &>> "${LOGFILE}"
-    readarray -t SERVER_MLNX <<< "$(ssh "${SERVER_IP}" "sudo mst status -v"  | awk '/mlx/{print $3 " " $4}' | sort -t ' ' -k2,2V)"
-    # Without CUDA
-    ngc_rdma_test
-    # Use CUDA
-    ngc_rdma_test "use_cuda"
+    echo "=== Server: ${SERVER_IP} ===" &> "${LOGFILE}"
```
Comment (blochl) [inline ngc_rdma_wrapper.sh:354]: No need for `&` here, as you have only stdout.

File: ngc_rdma_wrapper.sh [hunk 2]
```diff
@@ -178,14 +351,38 @@ if [[ $# == 1 ]]; then
...
+    ssh "${SERVER_IP}" sudo dmidecode -t 0,1 &>> "${LOGFILE}"
+    echo "=== Client: ${CLIENT_IP} ===" &>> "${LOGFILE}"
```
Comment (blochl) [inline ngc_rdma_wrapper.sh:356]: ...Same here.

File: ngc_internal_lb_rdma_wrapper.sh [hunk 3]
```diff
@@ -180,14 +180,16 @@ nic_to_gpu_affinity() {
             echo "NIC to GPU affinity according to ${AFFINITY_FILE} file:" | tee -a "${LOGFILE}"
             GPU_LINE=$(grep -ni "gpu" "${AFFINITY_FILE}" | cut -d ':' -f1)
             NIC_LINE=$(grep -ni "mlx" "${AFFINITY_FILE}" | cut -d ':' -f1)
+            if [[ -z $NIC_LINE || -z $GPU_LINE ]]; then
```
[the remark this answers is not in the document]
Comment (RavidRothfeld) [reply ngc_internal_lb_rdma_wrapper.sh]: Fixed :)

File: ngc_rdma_wrapper.sh (hunk 1 above)
Comment (RavidRothfeld) [reply ngc_rdma_wrapper.sh]: Removed.

File: ngc_rdma_wrapper.sh (hunk 2 above)
Comment (RavidRothfeld) [reply ngc_rdma_wrapper.sh]: Removed.

**Review by blochl — APPROVED** [review]

**Pull request merged**